### ISSUE TITLE
[api-extractor] Exported `let` from namespace ends up in types

### ISSUE THREAD

**Issue by nvh** [Microsoft/web-build-tools#976]
Input:
```typescript
export namespace Data {
    /** @internal */
    export let variable: object[] = []
}
```
Output:
```typescript
export declare namespace Data {
    /** @internal */
    let /* Excluded from this release type: variable */;
}
```

Expected output:
```typescript
export declare namespace Data {
    /* Excluded from this release type: variable */;
}
```

**Comment by nvh** [comment]
It is for both versions, actually. And I notice that the same goes for `const`

**Comment by pgonzal** [comment]
Yeah, this is a bug

**Comment by pgonzal** [comment]
This was fixed in **api-extractor** beta version 7.0.2.
